packages/fx-core/src/plugins/resource/sql/managementClient.ts

```diff
@@ -217,13 +218,15 @@ export class ManagementClient {
       startIpAddress: startIp,
       endIpAddress: endIp,
     };
+    const ruleName = Constants.firewall.localRule + this.nextIndex;
```

Comment:
for the rule name can we have a extracted method to get rule name so that it could be shared between add and delete to avoid future regression?

Reply:
OK.
Added a specific method getRuleName to get the rule name. Thanks.